tensorflow/lite/kernels/internal/optimized/optimized_ops.h

```diff
@@ -2351,6 +2349,24 @@ inline void Relu(const RuntimeShape& input_shape, const float* input_data,
   output = input.cwiseMax(0.0f);
 }
 
+inline void Relu1(const RuntimeShape& input_shape, const float* input_data,
+                  const RuntimeShape& output_shape, float* output_data) {
+  gemmlowp::ScopedProfilingLabel label("Relu1 (not fused)");
+
+  const auto input = MapAsVector(input_data, input_shape);
+  auto output = MapAsVector(output_data, output_shape);
+  output = input.cwiseMax(-1.0f).cwiseMin(1.0f);
```

Comment:
Is this faster than a clamp? Or does it do the same thing? Same for the other activation.